sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/text/TextSuite.scala

```diff
@@ -33,8 +33,8 @@ class TextSuite extends QueryTest with SharedSQLContext {
     verifyFrame(sqlContext.read.text(testFile))
   }
 
-  test("writing") {
-    val df = sqlContext.read.text(testFile)
+  test("SPARK-12562 verify write.text() can handle column name beyond `value`") {
+    val df = sqlContext.read.text(testFile).withColumnRenamed("value", "adwrasdf")
```

Comment:
After `write.text()`, the local text file actually does not carry the schema name like JSON does. When reading back the text file and then call `verifyFrame`, it will always have `value` as the column name.